### PR TITLE
Convert segmentaion fault in TDE library to user error

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/COM-703

Changes:
- Segmentation fault in Python TDE library is now user error (we cannot fix it).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM php:7.4
 # setup the python environment
 WORKDIR /tmp
 RUN apt-get update && apt-get install -y wget git tar python unzip
-RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN wget https://bootstrap.pypa.io/pip/2.7/get-pip.py
```
Comment (webrouse) [inline Dockerfile:6]: Pouziva sa stary python, povodna url ide uz iba s novym.

File: Dockerfile [hunk 2]
```diff
@@ -30,4 +30,4 @@ RUN composer install --no-interaction
 
 WORKDIR /home
 RUN du -sh *
-CMD python -u ./src/main.py --data=/data
+CMD ./src/run.sh
```
Comment (webrouse) [inline Dockerfile:33]: Segmentation fault je fatal error, neda sa v Pythone odchytit, preto je to obalene do bash scriptu.

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,6 @@
 DATADIR=$(PWD)/ignored/data
 TESTDATADIR=$(PWD)/testdata
+TESTDATADIR_SEG_FAIL=$(PWD)/testdata-seg-fault
```
Comment (webrouse) [inline Makefile:3]: Test som pridal do Makefile, kedze sa to musi testovat z najvyssej urovne (aby sa spustil `run.sh`)

File: testdata-seg-fault/config.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+parameters:
+  tables:
+    in.c-ex-dummy.dummy:
+      tdename: in.c_ex_dummy.dummy.tde
+  typedefs:
+    in.c-ex-dummy.dummy:
+      col1:
+        type: string
+      col2:
+        type: date
```
Comment (webrouse) [inline testdata-seg-fault/config.yml:11]: Test: v produkcii to pada vacsinou na nevalidnom datume.

File: testdata-seg-fault/in/tables/in.c-ex-dummy.dummy [hunk 5]
```diff
@@ -0,0 +1,2 @@
+"col1","col2"
+"abc","0020-02-18"
```
Comment (webrouse) [inline testdata-seg-fault/in/tables/in.c-ex-dummy.dummy:2]: Z nejakeho dovodu TDE knicnica nevie parsnut datum `0020-02-18`. 
Je to 6 rokov stara binarka, ... mame task na novu verziu, ... ale do logov chodi vela app. errors, tak to musime nejak fixnut aj tu.

**Review by ondrajodas — APPROVED** [review]
ok

**Pull request merged**